src/ensembl/src/content/app/browser/browser-image/BrowserImage.test.tsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import { Provider } from 'react-redux';
+import { mount } from 'enzyme';
+
+import { BrowserImage, BrowserImageProps } from './BrowserImage';
+import { CircleLoader } from 'src/shared/components/loader/Loader';
+import configureStore from 'src/store';
+
+describe('<BrowserImage />', () => {
+  afterEach(() => {
+    jest.resetAllMocks();
+  });
+
+  const defaultProps: BrowserImageProps = {
+    trackStates: {},
+    browserCogTrackList: {},
+    browserNavOpened: false,
+    regionEditorActive: false,
+    regionFieldActive: false,
+    browserActivated: false,
+    activateBrowser: jest.fn(),
+    updateBrowserNavStates: jest.fn(),
+    updateBrowserActivated: jest.fn(),
+    updateBrowserActiveEnsObject: jest.fn(),
+    setChrLocation: jest.fn(),
+    setActualChrLocation: jest.fn(),
+    updateMessageCounter: jest.fn(),
+    updateDefaultPositionFlag: jest.fn(),
+    changeHighlightedTrackId: jest.fn()
+  };
+
+  const store = configureStore();
+
+  const wrappingComponent = (props: any) => (
+    <Provider store={store}>{props.children}</Provider>
```

Comment:
If Provider was used so that connected nested components could properly render, maybe mock these components while importing them?